### PR TITLE
Release v5.2

### PR DESCRIPTION
**Pull request by kwoodson**
```release-note
NONE
```
Bumping release to v5.2.

### CODE REVIEW

**Comment by ehashman** [comment]
Did we have any clusters running 5.1?

**Comment by jim-minter** [comment]
@kwoodson check the unit test failure
@mjudeikis please can you prioritise this tomorrow am?

File: pkg/arm/arm.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func New(ctx context.Context, log *logrus.Entry, cs *api.OpenShiftManagedCluster
 		return v3.New(ctx, log, cs, testConfig), nil
 	case "v4.2", "v4.3", "v4.4":
 		return v4.New(ctx, log, cs, testConfig), nil
-	case "v5.1":
```
Comment (mjudeikis) [inline pkg/arm/arm.go]: we skipping 5.1 and going straight to 5.2

**Pull request merged**